### PR TITLE
Story/x1154: Fields renamed according to u_ and x_ convention.

### PR DESCRIPTION
**Pull request by armandcela**

### CODE REVIEW

**Review by LoKKeR96 — APPROVED** [review]

**Comment by PeterC10** [comment]
Can we add the prefix `display_name`, `parent_id` (can also update `_parent_name`) and child/children_ids?

File: addons/package_hierarchy/models/stock_quant_package.py [hunk 1]
```diff
@@ -38,46 +37,47 @@ class QuantPackage(models.Model):
             "and within the contained package hierarchy."
         ),
     )
-    children_ids = fields.One2many("stock.quant.package", "parent_id", "Contained Packages")
-    depth = fields.Integer(compute="_compute_depth", store=True)
+    # Renaming to child_ids as from hierarchy tree view should have child_ids for the new tree view
```
Comment (PeterC10) [inline addons/package_hierarchy/models/stock_quant_package.py:40]: This view is added by a closed source module, we shouldn't be referencing it here.

As said in the other PR, we should discuss this more, but renaming our fields to work around a view doesn't feel right.

If we go by what the story is for then the field should be `x_children_ids` (unless `child_ids` is a reserved field name in Odoo, I can't remember)

Comment (armandcela) [reply addons/package_hierarchy/models/stock_quant_package.py:40]: Removed the comment :)

**Comment by armandcela** [comment]
> Can we add the prefix `display_name`, `parent_id` (can also update `_parent_name`) and child/children_ids?

These are odoo special fields, I think we can agree on these fields to be added without convention (lets make exception for these fields).
display_name is on Basic fields of get_info method
parent_id and child_ids are special fields, i think would be better to agree on these fields not to add by convention

**Comment by PeterC10** [comment]
> > Can we add the prefix `display_name`, `parent_id` (can also update `_parent_name`) and child/children_ids?
> 
> These are odoo special fields, I think we can agree on these fields to be added without convention (lets make exception for these fields). display_name is on Basic fields of get_info method parent_id and child_ids are special fields, i think would be better to agree on these fields not to add by convention

Yes, so this is where it gets awkward with our convention and having Package Hierarchy outside UDES Open.

Happy with how it is, but just to explain for others for full visibility:

1. `display_name` is used in `get_info` in UDES Open, but this module is outside the scope of UDES Open, so it shouldn't be relevant but happy to leave it.
2. `parent_id` is the default `_parent_name` value on every model, but can be changed to anything, like `location_id` on `stock.location`. But we are saying now for our convention that `parent_id` and `child_ids` can be excluded from this, as it is frankly just easier to have a consistent naming across as many models as possible.

**Review by PeterC10 — APPROVED** [review]

**Pull request merged**